### DISCUSSION TITLE
CCTV-1好像没法命中epg中的名字

### DISCUSSION THREAD

**Discussion by nankaine** [kimwang1978/collect-tv-txt#3] [General]
我找了几个EPG，名字都没有中间的短杠。
@kimwang1978

**Comment by kimwang1978** [comment]
> 我找了几个EPG，名字都没有中间的短杠。 @kimwang1978

把中间的横杠去掉了，再看看EPG能不能匹配上。

**Reply by nankaine** [reply]
> > 我找了几个EPG，名字都没有中间的短杠。 @kimwang1978
> 
> 把中间的横杠去掉了，再看看EPG能不能匹配上。

![image](https://github.com/kimwang1978/collect-tv-txt/assets/94109329/e4472db0-0554-4005-8289-011af6008c28)
![image](https://github.com/kimwang1978/collect-tv-txt/assets/94109329/071d4bfd-99e6-415a-944d-fee63b3ae357)


目前看着能正常匹配上

**Reply by kimwang1978** [reply]
你上面这个播放器是potplayer？ @nankaine

**Reply by nankaine** [reply]
> 你上面这个播放器是potplayer？ @nankaine

黑鸟播放器 @kimwang1978
